### PR TITLE
Installation README Update

### PR DESCRIPTION
**Pull request by 0x11DFE**
Since the update 0.4.0 the installation process has changed and so the readme usage is updated accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (SpriteOvO) [inline README.md:1]: It doesn't seem to make any sense, it should be reverted.

File: README.md [hunk 2]
```diff
@@ -20,22 +21,33 @@ Telegram local messages anti-revoke plugin.
 
 ## :tomato: Compatibility
 Usually the plugin is not affected by small updates of Telegram.  
-But some updates may cause the plugin to error or crash, please [report the issue](https://github.com/SpriteOvO/Telegram-Anti-Revoke/issues) and wait for the plugin to be updated.
+But some updates may cause the plugin to error or crash, please [report the issue](/issues) and wait for the plugin to be updated.
```
Comment (SpriteOvO) [inline README.md:24]: Relative links in `README.md` are relative to the repository directory, this change will cause it to jump to `https://github.com/SpriteOvO/Telegram-Anti-Revoke/blob/dev/issues`, it should be reverted.

File: README.md [hunk 3]
```diff
@@ -20,22 +21,33 @@ Telegram local messages anti-revoke plugin.
...
 
 If you are using **Telegram beta**, please check the latest preview version.  
 (Compatibility updates for **Telegram beta** may not be released in time)
 
 ## :hamburger: Usage
-1. Go to the [Release Page](https://github.com/SpriteOvO/Telegram-Anti-Revoke/releases) to download the latest version of the plugin.  
-2. Close the running Telegram process.  
-3. Copy *version.dll* file into the Telegram root directory.  
-4. Run the Telegram program.
+1. Go to the [Release Page](/releases) to download the latest version of the plugin.
```
Comment (SpriteOvO) [inline README.md:30]: The `Release Page` also needs to be reverted.

File: README.md [hunk 4]
```diff
@@ -20,22 +21,33 @@ Telegram local messages anti-revoke plugin.
...
 If you are using **Telegram beta**, please check the latest preview version.  
 (Compatibility updates for **Telegram beta** may not be released in time)
 
 ## :hamburger: Usage
-1. Go to the [Release Page](https://github.com/SpriteOvO/Telegram-Anti-Revoke/releases) to download the latest version of the plugin.  
-2. Close the running Telegram process.  
-3. Copy *version.dll* file into the Telegram root directory.  
-4. Run the Telegram program.
+1. Go to the [Release Page](/releases) to download the latest version of the plugin.  
+2. Open Telegram and open the top-left menu. From there, note the application version and architecture \***ARCH**\* (x86 or x64) which is labelled under its version, e.g "**Version** x.x.x **x64**".
```
Comment (SpriteOvO) [inline README.md:31]: If the user is using Telegram x86, only "Version x.x.x" will be displayed there.

File: README.md [hunk 5]
```diff
@@ -20,22 +21,33 @@ Telegram local messages anti-revoke plugin.
...
+
+### Before 2.8.5
+1. Move the file "TAR-Resources\\***ARCH**\*.dll" to the Telegram directory as `version.dll`.
+
+### After 2.8.4
+1. Turn off the option "Launch Telegram when system starts" in the Telegram settings.
+2. Move the file "TAR-Launcher-\***ARCH**\*.exe" and the folder "TAR-Resources" to the Telegram directory.
+3. Copy the file "TAR-Launcher-\***ARCH**\*.exe" to your clipboard.
+4. Navigate to `shell:startup` in Explorer `Win`+`R`).
+5. Right-click (open your context menu) and "Paste shortcut" in that directory.
```
Comment (SpriteOvO) [inline README.md:44]: auto-run is not required, so they should be separated and placed below.

File: README.md [hunk 6]
```diff
@@ -20,22 +21,33 @@ Telegram local messages anti-revoke plugin.
...
 
 ## :hammer_and_wrench: Build
 See the [Build Instructions](/Docs/Build.md).
 
 ## :bug: Report
-Please use the [Issue Tracker](https://github.com/SpriteOvO/Telegram-Anti-Revoke/issues) to report issues or suggest new features.  
+Please use the [Issue Tracker](/issues) to report issues or suggest new features.
```
Comment (SpriteOvO) [inline README.md:50]: Also needs to be reverted.

**Review by SpriteOvO — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (0x11DFE) [reply README.md]: Got it.

**Review by 0x11DFE — COMMENTED** [review]
fixes and addition suggested added.

**Review by SpriteOvO — APPROVED** [review]
Thanks!

**Pull request merged**